samples/bluetooth/peripheral_with_multiple_identities/README.rst

```diff
@@ -0,0 +1,190 @@
+
+How it works
+************
+
+Setting up the advertisers
+==========================
+
+The sample application will setup and start as many advertisers as specified by **CONFIG_BT_EXT_ADV_MAX_ADV_SET**.
+The :c:func:`setup_advertiser()` configures and initialises a connectable advertiser with a unique name and identity:
+
+#. Identities are created with the API :c:func:`bt_id_create()`.
+#. To assign a given identity to an advertiser, the field :c:member:`bt_le_adv_param.id` is set.
+#. A unique name is then generated for each advertiser consiting of **CONFIG_BT_DEVICE_NAME** value followed by the identity.
+#. The APIs :c:func:`bt_le_ext_adv_create()` and :c:func:`bt_le_ext_adv_set_data()` will create the advertising set and set its advertising data, respectively.
+
+Each advertiser is setup as a connectable legacy advertiser but the extended advertising version of the API is used as it supports multiple advertising instances and sets. 
+
+A work item is then initialized for each advertiser, in order to avoid race conditions and ensure that tasks are executed independently.
+Each work items handler function is set to :c:func:`start_connectable_advertiser()` which will start the advertiser associated with the work item.
+This is done by calling :c:func:`bt_le_ext_adv_start()`. Note: By default this function will not attempt to resume connectable advertising after a connection is established. 
+Each advertisers work item is then submitted to the workqueue to handle the starting of each advertiser asynchronously.
+The sample will then continue executing asynchronously using callbacks to define its behavior.
+
+Connection callbacks
+====================
+
+Two connection callbacks are then utilized (see :c:struct:`bt_conn_cb`):
+
+#. **Connected callback:** When a connection is established the sample will print the central's address. 
+#. **Disconnected callback:** When a disconnection occurs the central's address is printed along with the advertiser that was disconnected.
+   The identity associated with the connection is retrieved by calling :c:func:`bt_conn_get_info()` to obtain :c:member:`bt_conn_info.id`.
+   The work item (advertiser) associated with this identity is then submitted to the workqueue to restart the advertiser. 
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/peripheral_with_multiple_identities`
+
+.. include:: /includes/build_and_run_ns.txt
+
+Testing
+=======
+
+|test_sample|
+
+1. |connect_kit|
+#. |connect_terminal|
+#. Start the `nRF Connect for Mobile`_ application on your smartphone or tablet.
+
+   There should now be multiple devices advertising with the name ``Nordic multi adv id`` followed by the ID.
+#. Connect to the device and observe the central's address is printed.
+#. Disconnect from any device and observe that the correct advertiser is disconnected.
+#. The disconnected advertiser should then be restarted.
```

Comment:
```suggestion
#. The disconnected advertiser is restarted.
```
Is this a step (like Restart the disconnected advertiser) or something that happens as a result of the previous step? If it is the result, then maybe like I suggested.

Reply:
It is a result :) So I think you are right.